src/state/dune

```diff
@@ -20,6 +20,7 @@
  (modules Learnocaml_version Learnocaml_api)
  (libraries ocplib-json-typed
             ezjsonm
+	    conduit
```

Comment:
What is `conduit`?

Reply:
It is the library used by `cohttp` to abstract connections.